src/avs/AVSDetailsTVLTab.jsx

```diff
@@ -0,0 +1,273 @@
+      }}
+      topContent={
+        <div className="text-foreground-1 text-medium">LST distribution</div>
+      }
+    >
+      <TableHeader>
+        <TableColumn className="w-1/2">LST</TableColumn>
+        <TableColumn>ETH</TableColumn>
+      </TableHeader>
+      <TableBody>
+        {isAVSLoading &&
+          [...new Array(12)].map((_, i) => (
+            <TableRow key={i}>
+              <TableCell className="pl-0">
+                <Skeleton className="h-5 w-full rounded-md" />
+              </TableCell>
+              <TableCell className="flex justify-end">
+                <Skeleton className="h-10 w-20 rounded-md" />
+              </TableCell>
+            </TableRow>
+          ))}
+        {!isAVSLoading &&
+          sortedTokens.map(([key, total]) => (
+            <TableRow key={key}>
+              <TableCell className="text-sm pl-0">
+                <div className="flex items-center gap-x-2">
+                  <Image
+                    height={16}
+                    width={16}
+                    src={STRATEGY_ASSET_MAPPING[key].logo}
+                    fallbackSrc="/eth.png"
+                  />
+                  <span className="text-foreground-2 truncate">
+                    {STRATEGY_ASSET_MAPPING[key]?.name}
+                  </span>{' '}
+                  <span className="text-foreground-1">
+                    {STRATEGY_ASSET_MAPPING[key]?.symbol}
+                  </span>
+                </div>
+              </TableCell>
+              <TableCell className="flex justify-end">
+                <div className="text-end">
+                  <div>{formatUSD(Number(total) * ethRate, compact)}</div>
+
+                  <div className="text-xs text-foreground-2">
+                    {formatETH(Number(total), compact)}
+                  </div>
+                </div>
+              </TableCell>
+            </TableRow>
+          ))}
+      </TableBody>
+    </Table>
+  );
+}
+
+function EigenDisclaimer() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <div className="inline-flex gap-x-1 items-center">
+      N/A
+      <Tooltip
+        showArrow={true}
+        placement="top"
+        content={
+          <div className="max-w-[250px] p-4 break-words">
+            <div className="text-sm">
+              EIGEN is currently not listed on any exchanges so we are unable to
+              get its USD value. Information will be updated when the token is
+              available on centralized/decentralized exchanges.
+            </div>
+          </div>
+        }
+        isOpen={isOpen}
+      >
+        <span
+          className="text-sm material-symbols-outlined cursor-pointer"
+          style={{
+            fontVariationSettings: `'FILL' 0`
+          }}
+          onMouseEnter={() => setIsOpen(true)}
+          onMouseLeave={() => setIsOpen(false)}
+          // to make tooltip work on mobile
+          onPress={() => setIsOpen(!isOpen)}
```

Comment:
Although `onPress` works, it throws an error in console:
> Warning: Unknown event handler property `onPress`. It will be ignored.

The modern approach is:

```suggestion
          onPointerDown={() => setIsOpen(!isOpen)}
```